ci/kind/kind-setup.sh

```diff
@@ -244,25 +246,68 @@ function configure_extra_networks {
   done
 }
 
+function get_bridge_id {
```

Comment:
I wonder why it needs to extract this function, does previous code have any issue?

Reply:
I was trying to handle the case if there is no kind network id, but I think it's already handled by other network functions, I removed that function.